src/bp/ui-admin/src/telemetry/client.ts

```diff
@@ -0,0 +1,225 @@
+import axios from 'axios'
+import shared from 'botpress/shared'
+import { createHash } from 'crypto'
+import _ from 'lodash'
+import ms from 'ms'
+import uuid from 'uuid'
+
+import store from '../store'
+
+export interface StoreInfoBody {
+  storedInfo: string
+  loadInfo: Function
+}
+
+export interface StoreInfoType {
+  [key: string]: StoreInfoBody
+}
+
+export const storeInfos: StoreInfoType = {}
```

Comment:
Why exporting an interface and using a setter?

Reply:
I'm not sure I understand? Is the problem the interface or the setter or the combination of both of them? I just want to be sure

Reply:
No inherent problem, I just thought it was a bit much having all that code for what is basically a hard-coded constant